session/sessiontest/config.go

```diff
@@ -122,8 +122,8 @@ func NewConfig(rng *rand.Rand, contacts ...perun.Peer) (session.Config, error) {
 		DatabaseDir:       databaseDir,
 		PeerReconnTimeout: 20 * time.Second,
 
-		ContactsType: "yaml",
-		ContactsURL:  contactsYAMLFile,
+		IDProviderType: "yaml",
```

Comment:
```suggestion
		IDProviderType: "local",
```

ID provider type should be `local`, instead of `yaml`.

Reply:
This specifies the type of file that is "yaml", else this will make issue in opening a session.